catalog/ui/search-ui/standard/src/main/webapp/js/widgets/cesium.mapclustering.js

```diff
@@ -0,0 +1,187 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/* global define */
+define(['backbone',
+        'cesium',
+        'jquery'
+], function(Backbone, Cesium, $) {
+
+    var DEFAULT_PIXEL_DISTANCE = 100;
+
+    var MAX_RADIUS_DISTANCE = 2500000;
+
+    var MIN_CAMERA_HEIGHT = 5;
+
+    var MARKER_COLOR = Cesium.Color.RED;
+
+    var MapClustering = {};
+
+    var clusters = [];
```

Comment:
Curious why you put all this stuff out here? You aren't really using the model as a model. As it is now, it is a controller. That isn't necessarily a bad thing, but if you're going to stick with the controller, you shouldn't be extending Backbone.Model.